README.md

```diff
@@ -24,7 +24,7 @@ var databasePlugin = require('./some-database')
 var bluetoothPlugin = require('./bluetooth')
 var config = require('./some-config')
 
-var App = SecretStack({ appKey: '1KHLiKZvAvjbY1ziZEHMXawbCEIM6qwjCDm3VYRan/s=' })
+var App = SecretStack({ global: { appKey: '1KHLiKZvAvjbY1ziZEHMXawbCEIM6qwjCDm3VYRan/s=' } })
```

Comment:
> I have never used appKey. I would be in favoure of deprecating (wasn't it replaced with caps?)

Out of scope?

> also don't think "global" is needed here? It feels more implicit to me that these are global?

It's necessary in case you want to specify *at this point* some plugin-specific config, then you need a way of saying that the other configs are global, not plugin-specific. It can also be generally less confusing, if both `SecretStack({ ... })` and `.create(null, { ... })` object shapes follow the same schema.